.github/workflows/ci.yaml

```diff
@@ -56,6 +56,30 @@
       - name: Test
         run: |
           yarn nx affected --target=test --parallel=3 --ci --code-coverage
+  
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+
+    env:
+      NX_SKIP_NX_CACHE: true
+      
+    steps:
+      - name: Checkout
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
```

Comment:
For `actions/setup-node` it is up to v4 at this point, do you think this would be a good opportunity to bump this version number up while also pinning it to a hash instead of the version number way? @michael-valdron 

https://github.com/actions/setup-node/tags

Reply:
> For actions/setup-node it is up to v4 at this point, do you think this would be a good opportunity to bump this version number up while also pinning it to a hash instead of the version number way? @michael-valdron

@Jdubrick Shouldn't hurt to try updating it to v4 as well, could always revert it to v3 with pinning if the checks break.